### ISSUE TITLE
a question about the hyperparame

### ISSUE THREAD

**Issue by charlotte0104** [luozhenjie1997/CPL-Diff#1]
Thank you very much for your work, which has been of great help to me. The saved model  you provided in the project have achieved very good results. However, when I retrained the model using the hyperparameters you provided, gradient explosion occurred. Although I avoided gradient explosion by reducing the learning rate, I still could not obtain results as good as yours. Could you please share the detailed hyperparameters you used during training?

**Comment by luozhenjie1997** [comment]
Thank you very much for recognizing our work! After reviewing it, we found that there was an error in the code we uploaded, and we have now corrected it. You can re-clone our code repository and retrain the model. Once again, thank you for your recognition of our work!
